src/pages/posts/es/[slug].astro

```diff
@@ -6,13 +6,13 @@ import { Post } from '../../../types';
 
```

Comment:
by the way, i'm not totally sure we need two different templates for languages... the `DualPost` type i came up with supplies each language-specific type with a `lang: Lang` property, which can be used for conditional logic.  worth thinking about at least.